plugins/syntax_dev_legacy.py

```diff
@@ -255,7 +258,7 @@ def run(self, edit,
 
             if not text:
                 output.print("Error re-dumping the data in file (no output).")
-                status("Error re-dumping the data (no output).", True)
+                status("Error re-dumping the data (no output).", window, True)
```

Comment:
You updated all callsites of this function but didn't change the function itself.

Reply:
I left out the `completions.py` as well. Hopefully, everything is resolved.